src/clims/models/substance.py

```diff
@@ -17,39 +18,38 @@ class Substance(Model):
     NOTE: Use the substances service class (`from clims.services import substances`) to create or
     update substances and properties as otherwise business rules will be broken, e.g. you might
     update a substance but not its properties.
-
-    NOTE: It's currently not supported to change the original name of the substance, but a plugin
-    can specify a versioned name property that can be changed.
     """
     __core__ = True
 
     def __init__(self, *args, **kwargs):
         super(Substance, self).__init__(*args, **kwargs)
 
-    name = models.TextField()
-
-    version = models.IntegerField(default=1)
+    name = models.TextField(null=True)  # TODO: null=False
```

Comment:
It's not :) This should have been marked with # NOMERGE (which I do when I want to fix it before I send the PR).

It can sometimes be convenient to allow nulls while dealing with the django migrations. This is not an obvious case of that. Assume some strange stuff was happening.

Reply:
... so I'm fixing this.